src/util/accessibility.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Add accessibility features to help support announcing events and content changes for screen readers.
+ *
+ * @since 5.2.0
+ */
+
+/**
+ * Create the announcement element
+ */
+export default function createAnnouncementElement() {
+    var announcement = document.getElementById("announce-waveform");
```

Comment:
Thanks for your solution!
In general I see couple of issue with it:
- We can use `const` instead of `var`
- Do not repeat yourself with `document.getElementById("announce-waveform")`
- Use existing style utils for styling, to not repeat `Object.keys(styles).forEach..`

Also I think the number of `document.getElementById("announce-waveform")` calls, which are quite expensive for the performance. I'd suggest to use class instance or closures. Since almost everything in the project is done by using classes, I'd go for it:

```
import style from './style';

export class Accessibility {
  constructor() {
    this.announcement = document.body.appendChild(
      document.createElement('div')
    );
    this.announcement.setAttribute('aria-live', 'polite');

    style(this.announcement, {
      position: 'absolute',
      width: '1px',
      height: '1px',
      ...other,
    });

    this.updateAnnouncementText('Loading waveform');
  }

  updateAnnouncementText(text) {
    this.announcement.textContent = text;
  }

  clearAnnouncementText() {
    this.announcement.textContent = '';
  }
}
```

And then in WaveSurfer class:

```
class WaveSurfer {
  constructor() {
    this.accessibility = new Accessibility();
  }

  play() {
    this.accessibility.updateAnnouncementText('audio playing');
  }
}
```

Reply:
yea using classes is better, I'll update it